roles/splunk/defaults/main.yml

```diff
@@ -75,3 +75,15 @@ add_pstack_script: false # Set to true to install a pstack generation script for
 configure_dmesg: false
 install_utilities: false # Set to true to install the list of packages defined in the linux_packages var after installing splunk
 use_tuned_thp: false
+# Firewall configs
+configure_firewall: false # Whether or not to configure the firewall service on your machine, if set to true, opens firewall ports using UFW (default) or Firewalld depending on OS
+# Firewall port presets - reference these in group_vars to assign them to splunk
+splunkweb_port: {protocol: "tcp", number: 8000}
+splunkhec_port: {protocol: "tcp", number: 8088}
+splunktcpin_port: {protocol: "tcp", number: 9997}
+splunkapi_port: {protocol: "tcp", number: "{{ splunkd_port }}"}
```

Comment:
I guess `splunkapi_port` is more descriptive of what it's actually for?

Reply:
I think so, and it's distinct from the format to include as a dictionary